policies/policies.go

```diff
@@ -66,8 +66,10 @@ type MetricsSource interface {
 	MetricImplementation(lang assessment.MetricImplementation_Language, metric string) (*assessment.MetricImplementation, error)
 }
 
-func createKey(types []string) (key string) {
-	key = strings.Join(types, "-")
+// createKey creates a key by concatenating toolID and all types
+func createKey(toolID string, types []string) (key string) {
+	// Merge toolID and types to one slice and concatenate all its elements
+	key = strings.Join(append(types, toolID), "-")
 	key = strings.ReplaceAll(key, " ", "")
```

Comment:
Does the key (types and/or toolIds) really have spaces?

Reply:
Hopefully not. But it is not completely under our control.